src/systemcmds/pwm/pwm.c

```diff
@@ -122,7 +122,7 @@ get_parameter_value(const char *option, const char *paramDescription)
 	if (strncmp("p:", option, 2) == 0) {
 
 		char paramName[32];
-		strncpy(paramName, option + 2, 16);
+		strncpy(paramName, option + 2, 17);
```

Comment:
I knew that exactly 17 bytes should be written to paramName, but I couldn't find any guarantee that option points to a string with length < 17 
Interesting that PWM_AUX_DISARMED is 16 bytes long.